src/api/PDFEmbeddedFile.ts

```diff
@@ -0,0 +1,55 @@
+import Embeddable from 'src/api/Embeddable';
+import { PDFDocument } from '.';
+import PDFAttachmentEmbedder from 'src/core/embedders/PDFAttachmentEmbedder';
+import { PDFName, PDFArray, PDFDict, PDFHexString } from 'src/core';
+
+export default class PDFEmbeddedFile implements Embeddable {
+  static of = (doc: PDFDocument, embedder: PDFAttachmentEmbedder) =>
```

Comment:
Can we update this model to accept a `ref` like the other embeddables? e.g. [`PDFEmbeddedPage`](https://github.com/Hopding/pdf-lib/blob/master/src/api/PDFEmbeddedPage.ts#L25-L26). We'll also want to update `PDFEmbeddedFile.embed()` and `PDFAttachmentEmbedder.embedIntoContext` to accept/use the `ref` (you can reference any of the others embedders to see how this should work).

Reply:
Didn't understand was `this.context.nextRef()` was doing. My fault.